content/docs/APIReference-MessengerContext.md

```diff
@@ -0,0 +1,997 @@
+---
+title: MessengerContext
+date: "2017-12-04"
+---
+
+* [Content Types](#content-types)
+* [Templates](#templates)
+* [Quick Replies](#quick-replies)
+* [Sender Actions](#sender-actions)
+* [Attachment Upload API](#attachment-upload-api)
+* [Tags](#tags)
+* [Message Batching](#message-batching)
```

Comment:
`Attachment Upload API`
`Tags`
`Message Batching`

these 3 links is not available